### PR TITLE
[develop2] Improve a bit the output msgs

### PR DESCRIPTION
**Pull request by memsharded**
Changelog: Feature: Improved global Conan output messages (create, install, export, etc.)
Docs: omit

### CODE REVIEW

File: conans/client/generators/__init__.py [hunk 1]
```diff
@@ -31,10 +31,11 @@ def _get_generator_class(generator_name):
     try:
         generator_class = _generators[generator_name]
         # This is identical to import ... form ... in terms of cacheing
-        return getattr(importlib.import_module(generator_class), generator_name)
     except KeyError as e:
         raise ConanException(f"Invalid generator '{generator_name}'. "
                              f"Available types: {', '.join(_generators)}") from e
+    try:
+        return getattr(importlib.import_module(generator_class), generator_name)
```
Comment (AbrilRBS) [inline conans/client/generators/__init__.py:38]: 👍

File: conan/tools/cmake/presets.py [hunk 2]
```diff
@@ -200,6 +208,7 @@ def write_cmake_presets(conanfile, toolchain_file, generator, cache_variables,
 
     preset_content = json.dumps(data, indent=4)
     save(preset_path, preset_content)
+    ConanOutput(str(conanfile)).info("CMakeToolchain generated: CMakePresets.json")
```
Comment (AbrilRBS) [inline conan/tools/cmake/presets.py:211]: I'm guessing these are `ConanFileInterface`s and that's why they can't just call `conanfile.output...`, right?

**Review by AbrilRBS — APPROVED** [review]
This looks so much nicer now!

**Review by czoido — APPROVED** [review]

File: conans/client/generators/__init__.py (hunk 1 above)
Comment (memsharded) [reply conans/client/generators/__init__.py:38]: yes, my IDE linter was warning against possible undefinition of variables, which might happen when a multi-line ``try`` clause can raise different exceptions in different lines. I try to remove these linter warnings when possible, as I have learned that often they can be a bug waiting for a corner case to become an issue.

File: conan/tools/cmake/presets.py (hunk 2 above)
Comment (memsharded) [reply conan/tools/cmake/presets.py:211]: No, good catch, I think this might be a full conanfile, this might be replaced by ``conanfile.output.info``

**Review by franramirez688 — APPROVED** [review]

**Pull request merged**